src/components/Label/Label.tsx

```diff
@@ -0,0 +1,25 @@
+import { LabelProps, Label as PatternflyLabel } from '@patternfly/react-core';
+
+import { EmptyStateSymbol } from 'components/EmptyStateSymbol/EmptyStateSymbol';
+
+interface ILabelMapperObject {
+  text: string;
+  color: LabelProps['color'];
+}
+
+export interface ILabelMapper {
+  [key: string]: ILabelMapperObject;
+}
+
+interface ILabelProps {
+  labelObject: ILabelMapperObject | undefined;
+}
+
+/**
+ * Represents a Label generated by labelMapper object.
+ *
+ * @param labelObject - the mapper object that will be used to generate the label
+ */
+export const Label = ({ labelObject }: ILabelProps) => {
+  return labelObject ? <PatternflyLabel color={labelObject.color}>{labelObject.text}</PatternflyLabel> : <EmptyStateSymbol />;
```

Comment:
What's the situation when `labelObject` can be `undefined`? Should it be considered as error? If yes, then it should be handled like a error (at least uiLogger should log it).

Reply:
created a new comment